evap/results/tests/test_views.py

```diff
@@ -40,6 +40,7 @@ def setUpTestData(cls):
         student2 = mommy.make(UserProfile)
         cls.course = mommy.make(Course, id=21, state='published', semester=cls.semester, participants=[student1, student2], voters=[student1, student2])
         questionnaire = mommy.make(Questionnaire)
+        cls.contribution = mommy.make(Contribution, course=cls.course, questionnaires=[questionnaire], contributor=None)
```

Comment:
as @He3lixxx mentioned in a recent pr: shouldn't every course that has been saved already have a general contribution? can't you add the questionnaire to that?

Reply:
yeah, the code is currently plain wrong, since it adds a second contribution with contributor=None to the course.